volume/devicemapper/devicemapper.go

```diff
@@ -38,9 +53,11 @@ type DeviceMapperVolume struct {
 }
 
 // Init initializes the devicemapper driver
-func Init(root string) (volume.Driver, error) {
+func Init(root string, args []string, options map[string]string) (volume.Driver, error) {
```

Comment:
I think that each volume type should support its own argument parser in the least